<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>388</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Camera Calibration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>camera ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="CamID"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>height</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="height"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="x_range"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="y_range"/>
   </item>
   <item>
    <widget class="QPushButton" name="OK">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Calibration picture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="file_select_btn">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>select file</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
